### PR TITLE
Hamburger menu

### PR DESCRIPTION
**Pull request by digia3d**
When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,25 +33,25 @@
...
-        <a href="#">About</a>
+      <li class="nav-item">
+        <a href="#about" class="nav-link">About</a>
       </li>
-      <li>
-        <a href="#">Contact</a>
+      <li class="nav-item">
+        <a href="#contact" class="nav-link">Contact</a>
       </li>
     </ul>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly check on your hamburger menu design, some elements are missing and the design doesn't match the design given, below is a comparison with the Figma design.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

File: index.html [hunk 2]
```diff
@@ -33,25 +33,25 @@
       <!--</div>-->
     </div>
```
Comment (danmainah) [inline index.html:64]: - [ ] [optional] Kindly check your design in your header for both mobile and desktop versions, some elements are not well aligned.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @digia3d  and @bienvenuushindi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @digia3d  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️


- [OPTIONAL] Kindly review your headline section position, margins/paddings, sizing, etc. It is not aligned in the middle of the page and in mobile view some text is being cut out of the screen.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**